Sources/YPersistence/SortInfo.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SortInfo.swift
+//  YPersistence
+//
+//  Created by Sahil Saini on 17/04/23.
+//  Copyright © 2023 Y Media Labs. All rights reserved.
+//
+
+import Foundation
+
+/// Describes a mult-attribute (column) sort
+public struct SortInfo: Equatable {
+    /// Columns
+    public let columns: [SortColumn]
+
+    /// Converts the receiver to an array of sort descriptors.
+    public var descriptors: [NSSortDescriptor] {
+        columns.map { NSSortDescriptor(key: $0.key, ascending: $0.ascending) }
```

Comment:
```suggestion
        columns.map { $0.descriptor }
```
This was why we added `descriptor` to `SortColumn`.